### PR TITLE
Rename master api

### PR DESCRIPTION
**Pull request by maiqueb**
**What this PR does / why we need it**:
This PR renames the `master` attribute of the device plugin configuration to `lowerDevice` to foster inclusive language.

**Release note**:
```release-note
Rename the DP configuration `master` attribute to `lowerDevice`, thus fostering inclusive language.
```

### CODE REVIEW

File: pkg/deviceplugin/lister.go [hunk 1]
```diff
@@ -16,10 +16,10 @@ const (
 )
 
 type macvtapConfig struct {
-	Name     string `json:"name"`
-	Master   string `json:"master"`
-	Mode     string `json:"mode"`
-	Capacity int    `json:"capacity"`
+	Name        string `json:"name"`
+	LowerDevice string `json:"lowerDevice"`
```
Comment (phoracek) [inline pkg/deviceplugin/lister.go:20]: The key does not match your documentation changes where it was all lower case

**Review by phoracek — CHANGES_REQUESTED** [review]

File: pkg/deviceplugin/lister.go (hunk 1 above)
Comment (maiqueb) [reply pkg/deviceplugin/lister.go:20]: Done.

**Comment by jcaamano** [comment]
Thanks for the **cc**! I assume no backward compatibility concerns?

**Comment by maiqueb** [comment]
> Thanks for the **cc**! I assume no backward compatibility concerns?

I think no; we don't claim backwards compatibility anywhere. It'll be a bit painful to adjust whoever consumes this, but hey...

Best we can do is send an email to kubevirt-dev mailing list with an heads up... 

Your thoughts ?

**Comment by jcaamano** [comment]
That's ok. I just wanted to check merely if it had been considered.

**Review by phoracek — APPROVED** [review]

**Pull request merged**